### PR TITLE
Fix integration test.

### PR DESCRIPTION
**Pull request by vishh**

### CODE REVIEW

File: integration/framework.go [hunk 1]
```diff
@@ -39,6 +39,9 @@ type kubeFramework interface {
 
 	// Run kubectl.sh command
 	RunKubectlCmd(cmd ...string) (string, error)
+
+	// Cleanup cluster
+	Cleanup()
```
Comment (vmarmol) [inline integration/framework.go]: DestroyCluster?

**Comment by vmarmol** [comment]
Can we change the commit message to provide more detail about the change? Seems like it is cluster-destruction related.

**Comment by rjnagal** [comment]
btw, it requires credentials to view jenkins failure.

**Comment by vishh** [comment]
I haven't figured out the security model for jenkins yet. That is why its
restricted. I will create a user for you @rjnagal and @vmarmol with
temporary credentials.

File: integration/framework.go (hunk 1 above)
Comment (vishh) [inline integration/framework.go]: Done.

**Comment by rjnagal** [comment]
Merging, @vishh is still adding more fixes to the test.

**Comment by vishh** [comment]
@vmarmol: Addressed your comments.

**Pull request merged**